client/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "pisa-client",
-  "version": "0.1.2-rc1",
+  "version": "0.1.4-connext-beta.1",
```

Comment:
I don't think "connext" should be in the version number in master; forgot to remove?

Reply:
Bumped to 0.1.5